docs/variants/msi_z790/hardware-matrix.md

```diff
@@ -0,0 +1,31 @@
+# Hardware configuration matrix
+
+## Introduction
+
+This document describes the hardware configuration used for validation of the
+coreboot port on the MSI PRO Z790-P workstation. For additional information,
+please check the [FAQ section](./faq.md).
+
+## MSI PRO Z790-P, SN: 601-7E06-030B2210001398
+
+| Component                      | Description                                 |
+|--------------------------------|---------------------------------------------|
+| **CPU**                        | Intel Core i5-13600K 3.5                    |
+|                                | Noctua NH-U12S CPU Cooler                   |
```

Comment:
Something is off here. Why cooler is in the CPU section? Missing Fan/Cooling section maybe?

Reply:
I added CPU cooler section.

Reply:
OK, but what is this `3.5` in `Intel Core i5-13600K 3.5 ` ?

Reply:
Clock speed, it was also mentioned in other msi's configs co I added it accordingly.

Reply:
Then let's at least add GHz there... How one can guess what is 3.5?